build.gradle.kts

```diff
@@ -90,84 +91,82 @@ sourceSets {
 
 dependencies {
     implementation(kotlin("stdlib-jdk8"))
-    implementation("org.apache.commons:commons-lang3:3.14.0")
-    implementation("org.apache.commons:commons-text:1.11.0")
-    implementation("org.apache.commons:commons-collections4:4.4")
-    implementation("commons-io:commons-io:2.15.1")
-    implementation("com.opencsv:opencsv:5.9")
-    implementation("org.flowable:flowable-engine:7.0.1")
-    implementation("org.flowable:flowable-cdi:7.0.1")
-    implementation("org.flowable:flowable-cmmn-engine:7.0.1")
-    implementation("org.flowable:flowable-cmmn-cdi:7.0.1")
-    implementation("org.flowable:flowable-cmmn-engine-configurator:7.0.1")
-    implementation("org.slf4j:slf4j-jdk14:2.0.12")
-    implementation("com.auth0:java-jwt:4.4.0")
-    implementation("javax.cache:cache-api:1.1.1")
-    implementation("com.google.guava:guava:33.1.0-jre")
-    implementation("com.mailjet:mailjet-client:5.2.5")
-    implementation("com.itextpdf:kernel:8.0.3")
-    implementation("com.itextpdf:layout:8.0.3")
-    implementation("com.itextpdf:io:8.0.3")
-    implementation("com.itextpdf:html2pdf:5.0.3")
-    implementation("org.flywaydb:flyway-core:10.10.0")
-    implementation("org.flywaydb:flyway-database-postgresql:10.10.0")
-    implementation("org.apache.solr:solr-solrj:9.5.0")
-    implementation("nl.info.webdav:webdav-servlet:1.2.41")
-    implementation("net.sourceforge.htmlcleaner:htmlcleaner:2.29")
-    implementation("com.unboundid:unboundid-ldapsdk:7.0.0")
-
-    swaggerUI("org.webjars:swagger-ui:5.11.10")
+    implementation(libs.apache.commons.lang)
+    implementation(libs.apache.commons.text)
+    implementation(libs.apache.commons.collections)
+    implementation(libs.commons.io)
+    implementation(libs.opencsv)
+    implementation(libs.flowable.engine)
+    implementation(libs.flowable.cdi)
+    implementation(libs.flowable.cmmn.engine)
+    implementation(libs.flowable.cmmn.cdi)
+    implementation(libs.flowable.cmmn.engine.configurator)
+    implementation(libs.slf4j.jdk14)
+    implementation(libs.auth0.java.jwt)
+    implementation(libs.javax.cache.api)
+    implementation(libs.google.guava)
+    implementation(libs.mailjet.client)
+    implementation(libs.itextpdf.kernel)
+    implementation(libs.itextpdf.layout)
+    implementation(libs.itextpdf.io)
+    implementation(libs.itextpdf.html2pdf)
+    implementation(libs.flyway.core)
+    implementation(libs.flyway.postgresql)
+    implementation(libs.apache.solr)
+    implementation(libs.webdav.servlet)
+    implementation(libs.htmlcleaner)
+    implementation(libs.unboundid.ldapsdk)
```

Comment:
I think the Swagger UI dep is missing? Might be my bad from resolving merge conflicts on this PR. Not sure we still want that Swagger UI at all really but that's for another day..

Reply:
added this